nvm.sh

```diff
@@ -311,6 +311,12 @@ nvm_tree_contains_path() {
 
   local pathdir
   pathdir=$(dirname "${node_path}")
+
+  # get real directory in case of symbolic links 
+  if [ -d "${pathdir-}" ]; then
```

Comment:
actually since this is the output of `dirname`, we already know it's a dir - maybe this could be simplified (with line 313 above as well) to:
```sh
# get real directory in case of symbolic links
local pathdir
pathdir = "$(nvm_cd -P "$(dirname "${node_path}")" && pwd)"
```
?

Reply:
@ljharb It's up to you folks how to handle this.  I assumed that the directory might be not be valid, so I made the test, but perhaps there is previous validation?  FYI **dirname** does not verify a directory, just try this:

`dirname /xyz/abc/somefile.txt`

Reply:
ahhh true. hmm